opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -235,6 +235,16 @@ def test_start_span(self):
         span.start()
         self.assertEqual(start_time, span.start_time)
 
+        # default status
+        self.assertEqual(span.status.get_is_ok(), True)
+        self.assertEqual(span.status.get_canonical_code(), 0)
```

Comment:
We probably want to use `Enum` here. Refer to https://github.com/open-telemetry/opentelemetry-python/blob/b0da53d28b041338e04982f3a9715926887b5cad/opentelemetry-api/src/opentelemetry/trace/__init__.py#L115.